src/sentry/plugins/sentry_mail/activity/base.py

```diff
@@ -163,7 +163,7 @@ def avatar_as_html(self):
             return get_email_avatar(user.get_display_name(), user.get_label(), 20, True)
 
     def _get_sentry_avatar_url(self):
-        url = "/images/sentry-email-avatar.png"
+        url = "/images/email/sentry-email-avatar.png"
```

Comment:
If we move the image file we will break existing emails. That could be awkward for a few days before/after this change is deployed.

Reply:
Ohh ok - will revert